Add missing information for iOS (#181)
apache/cordova-plugin-geolocation: README.md

 This plugins requires the following usage description:

>  * `NSLocationWhenInUseUsageDescription` describes the reason that the app accesses the user's location, this used while the app is running in the foreground.

Reviewer: ```suggestion
 * `NSLocationWhenInUseUsageDescription` describes the reason that the app accesses the user's location while the app is running in the foreground.
```

>  * `NSLocationWhenInUseUsageDescription` describes the reason that the app accesses the user's location, this used while the app is running in the foreground.

Reviewer: ```suggestion
 * `NSLocationWhenInUseUsageDescription` describes the reason that the app accesses the user's location, this is used while the app is running in the foreground.
```

Reviewer: This suggestion is for grammar.

Reviewer 2: Seems good yes

>  * `NSLocationAlwaysAndWhenInUseUsageDescription` describes the reason that the app is requesting access to the user’s location information at all times, use this key if your iOS app accesses location information while running in the background and foreground. 

Reviewer: ```suggestion
 * `NSLocationAlwaysAndWhenInUseUsageDescription` describes the reason that the app is requesting access to the user’s location information at all times. Use this key if your iOS app accesses location information while running in the background and foreground. 
```

Reviewer: What do you think of this to break up a long-running sentence?

>  * `NSLocationAlwaysUsageDescription` describes the reason that the app is requesting access to the user's location at all times, use this key if your iOS app accesses location information in the background, and you deploy to a target earlier than iOS 11. In that case, add both this key and `NSLocationAlwaysAndWhenInUseUsageDescription` to your app’s `Info.plist` file with the same message.

Reviewer: ```suggestion
 * `NSLocationAlwaysUsageDescription` describes the reason that the app is requesting access to the user's location at all times. Use this key if your iOS app accesses location information in the background. and you deploy to a target earlier than iOS 11. In that case, add both this key and `NSLocationAlwaysAndWhenInUseUsageDescription` to your app’s `Info.plist` file with the same message.
```

> In that case, add both this key and `NSLocationAlwaysAndWhenInUseUsageDescription` to your app’s `Info.plist` file with the same message.

Can you clarify this? Are we suppose to use `NSLocationAlwaysAndWhenInUseUsageDescription` on iOS >= 12?

Reviewer 2: Hello,
Yes as apple mentioned in this link: https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationalwaysandwheninuseusagedescription

>  * `NSLocationAlwaysUsageDescription` describes the reason that the app is requesting access to the user's location at all times, use this key if your iOS app accesses location information in the background, and you deploy to a target earlier than iOS 11. In that case, add both this key and `NSLocationAlwaysAndWhenInUseUsageDescription` to your app’s `Info.plist` file with the same message.

Reviewer: ```suggestion
 * `NSLocationAlwaysUsageDescription` describes the reason that the app is requesting access to the user's location at all times. Use this key if your app accesses location information in the background and you deploy to a target earlier than iOS 11. For iOS 11 and later, add both `NSLocationAlwaysUsageDescription` and `NSLocationAlwaysAndWhenInUseUsageDescription` to your app’s `Info.plist` file with the same message.
```

Reviewer: This suggestion removes a run-on sentence and uses less "variables" such as "this" and "that" for clarity.

Reviewer 2: You are wrong, “earlier” mean < iOS 11  mean also older (not newer).

And also , this used is better than this is used in that sentence.

Regards

Reviewer: > “this used “ is better than “this is used “ in that sentence.

That sounds weird to me when speaking it out, but grammar check tells me that you are right and I am wrong... so you can ignore this suggestion.

According to https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationalwaysandwheninuseusagedescription

`NSLocationAlwaysAndWhenInUseUsageDescription` only applies to iOS 11+ and `NSLocationAlwaysUsageDescription` only applies to iOS 8-10

https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationalwaysusagedescription

Am I misunderstanding something?

Reviewer 2: Everything is good, thanks for info
